### DISCUSSION TITLE
Override AdminBundle Admin Login Template

### DISCUSSION THREAD

**Discussion by Humb3l** [pimcore/pimcore#9897] [Q&A / Support] [answered]
Hey Guys,
I'm trying to overwrite the Admin Login Template from the Pimcore Admin Bundle to disable the "forgot password" button. Currently I'm using Pimcore 6.9.6 and I tried to overwrite it, like it is described in the official Symfony Docs (https://symfony.com/doc/current/bundles/override.html) but it doesn't seem to work.

This is the structure I tried:
```
templates
└── bundles
    ├── AdminBundle
    │   └── Admin
    │       └── Login
    │           └── login.html.php
    └── PimcoreAdminBundle
        └── Admin
            └── Login
                └── login.html.php
```
I even tried to overwrite the template under app/resources but no success, I also tried to do this with Pimcore X but no success either.

I know there is already one discussion on Github about this topic (https://github.com/pimcore/pimcore/discussions/9204), but there has to been an easier way and also I'm not trying to extend the template.

I hope someone can help me figure out how to do this :)

**Comment by passioneight** [comment]
I took part in the discussion you referred to. I'm still not sure if there is any way other than high-jacking the response and altering it to suit your needs. Though, I'd love to see the guys from Pimcore providing something for this.

@dvesh3 maybe you have more information?

**Comment by dvesh3** [comment, marked as answer]
@Humb3l @passioneight It works for me on Pimcore X with following structure:
```
templates
└── bundles
    └── PimcoreAdminBundle
        └── Admin
            └── Login
                └── login.html.twig

```
on 6.9:
```
app
└── Resources
    └── PimcoreAdminBundle
        └── views
           └── Admin
                └── Login
                    └── login.html.php
```
**Important: please clear cache with command _php bin/console cache:clear_ after adding new template**
![image](https://user-images.githubusercontent.com/5137917/127624958-7b413f71-0bda-48c7-8e00-84c6c0fe2e48.png)

**Reply by Humb3l** [reply]
Thank you so much guys, it worked 🔥
